src/ui/public/filter_bar/__tests__/_add_filters.js

```diff
@@ -103,6 +103,29 @@ describe('add filters', function () {
       expect(globalState.filters.length).to.be(0);
     });
 
+    it('should allow overwriting a positive filter by a negated one', () => {
```

Comment:
It's not just negative filters that override positives ones is it? Any new filter will override an older one. Might not hurt to phrase it that way here, and add one more part to the the test to ensure a positive filter overrides a negative one.

Reply:
done